### PR TITLE
Add event tests

### PR DESCRIPTION
**Pull request by microbecode**
Add unit tests for testing event emission

### CODE REVIEW

**Comment by JaniAnttonen** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>74ed261</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://starknet-multisig-ekweafffv-equilibriumco.vercel.app'>https://starknet-multisig-ekweafffv-equilibriumco.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/equilibriumco/starknet-multisig/FF9EWrDo6sfH43PC1piWzvXyc3Q1'>https://vercel.com/equilibriumco/starknet-multisig/FF9EWrDo6sfH43PC1piWzvXyc3Q1</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/eqlabs/starknet-multisig/actions/runs/2567089009)

**Comment by JaniAnttonen** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>bfce777</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://starknet-multisig-pazyjx41x-equilibriumco.vercel.app'>https://starknet-multisig-pazyjx41x-equilibriumco.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/equilibriumco/starknet-multisig/ubfX438ahFvf1xtVJJ3FvY5W1xTm'>https://vercel.com/equilibriumco/starknet-multisig/ubfX438ahFvf1xtVJJ3FvY5W1xTm</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/eqlabs/starknet-multisig/actions/runs/2675125812)

**Comment by JaniAnttonen** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>85b2f63</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://starknet-multisig-7dgzpsrls-equilibriumco.vercel.app'>https://starknet-multisig-7dgzpsrls-equilibriumco.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/equilibriumco/starknet-multisig/DxXe2PkLN1nEPpbsSTPavVSS7ZLY'>https://vercel.com/equilibriumco/starknet-multisig/DxXe2PkLN1nEPpbsSTPavVSS7ZLY</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/eqlabs/starknet-multisig/actions/runs/2675134262)

File: contracts/test/Multisig.test.ts [hunk 1]
```diff
@@ -780,3 +873,29 @@ describe("Multisig with multiple owners", function () {
...
   });
 });
+
+const assertEvent = (
+  receipt: TransactionReceipt,
+  eventName: string,
+  eventData: IEventDataEntry[]
+) => {
+  const eventKey = getSelectorFromName(eventName);
+  const foundEvent = receipt.events.filter((e) =>
```
Comment (JaniAnttonen) [inline contracts/test/Multisig.test.ts:883]: Is the point here to actually find multiple events of the type or just one match? Could use .find() in that case.

**Review by JaniAnttonen — APPROVED** [review]
Approved with a comment. :)

File: contracts/test/Multisig.test.ts (hunk 1 above)
Comment (microbecode) [reply contracts/test/Multisig.test.ts:883]: Well, this way we make sure there is only one such event emitted. "find" wouldn't care if there are multiple. But good observation.

**Comment by JaniAnttonen** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>9162607</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://starknet-multisig-3qyjkmxby-equilibriumco.vercel.app'>https://starknet-multisig-3qyjkmxby-equilibriumco.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/equilibriumco/starknet-multisig/3SQkFfeTB2DN1GrPWB1ZrALTnSpm'>https://vercel.com/equilibriumco/starknet-multisig/3SQkFfeTB2DN1GrPWB1ZrALTnSpm</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/eqlabs/starknet-multisig/actions/runs/2676982815)

**Pull request merged**